apps/condo/domains/billing/schema/BillingIntegration.test.js

```diff
@@ -206,4 +207,42 @@ describe('BillingIntegration', () => {
             })
         })
     })
+
+    describe('Cache', async () => {
+        test('cache works with itemQuery', async () => {
```

Comment:
May be some more descriptive test names?)

Reply:
Done!